### DISCUSSION TITLE
Welcome to Speech Rule Engine Discussions!

### DISCUSSION THREAD

**Discussion by zorkow** [zorkow/speech-rule-engine#441] [General]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pkra** [comment]
:tada: 

Awesome idea.

**Comment by davidfarmer** [comment]
I want to devise a better way for people to write math.
Sort-of like TeX (but easier) and sort-of like AsciiMath
(but able to capture things like \begin{cases}...\end{cases}).

The main feature is that it will be easier to capture the
intent of the author, so that common constructions which
appear ambiguous when printed, are not ambiguous in the source.

Then the issue is to not lose the semantic information in the
pipeline from author's source to whatever format the speech rule
engine is reading.

I am optimistic about implementing something like this in PreTeXt.

Regards,

David Farmer
